index.js

```diff
@@ -20,7 +20,36 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+     return Recipe.create({ 
+      title: 'Chilli Con Carne',
+      level: 'Easy Peasy',
+      ingredients: 'minced beef, onions, garlic, tomatos, kidney beans',
+      cuisine: "Mexican",
+      dishType: 'main_course',
+      duration: 60,
+      creator: 'Ania',
+      created: ''
+    })
+      // .then(recipe => console.log(`Recipe title is: ${recipe.title}`, recipe))
+      // .catch(error => console.log('An error happened while saving a new recipe:', error));
+  })
+  .then(() => {
+    return Recipe.insertMany(data)
+      // .then(recipe => console.log(`Recipe title is: ${recipe.title}`, recipe))
+      // .catch(error => console.log('An error happened while saving a new recipe:', error));
+  })
+  .then(() => {
+    return Recipe.updateOne({title: 'Rigatoni alla Genovese'}, {duration: 100})
+      // .then(recipe => console.log(`Recipe duration updated`, recipe))
+      // .catch(error => console.log('An error happened while updating the recipe:', error));
+  })
+  .then(() => {
+    return Recipe.deleteOne({title: 'Carrot Cake'})
+      // .then(recipe => console.log(`Recipe deleted`, recipe))
+      // .catch(error => console.log('An error happened while deleting the recipe:', error));
+  })
+  .then(() => {
+    mongoose.connection.close();
   })
   .catch(error => {
     console.error('Error connecting to the database', error);
```

Comment:
You made it like a pro super Dirk. Personaly, I would just not do so many then() methods and do more Recipe.create(), Recipe.insertMany() and so on.

Good job man, I better watch out! :D